schemas/user_request.py

```diff
@@ -74,12 +74,12 @@ class UserRequestReviewIn(BaseModel):
     bed_place_id: int = None
 
     @validator('status_id')
-    def validate_status_id(cls, v):
-        if v not in [1,2]:
-            message = "Заяву можливо тільки ухвалити або відхилити."
-            raise ValueError(message)
+    def validate_status_id(cls, v):  # TODO may be there is need to use classmethod decorator
+        if v not in [1, 2]:
+            raise BackendException(message="The application can only be approved or rejected.")
```

Comment:
Don't use BackendException inside Pydantic schemas validation. it should be handled by another handler: validation_exception_handler.